dev/core/src/com/google/gwt/dev/util/arg/ArgHandlerWarDir.java

```diff
@@ -45,6 +45,11 @@ public String getTag() {
     return "-war";
   }
 
+  @Override
+  public String[] getTags() {
+    return new String[] {getTag(), "-launcherDir"};
```

Comment:
I can revert this if we don't think it makes sense, but it makes CodeServer and DevMode just a little more alike.